Assets/MixedRealityToolkit.Examples/Demos/Solvers/Scripts/SolverExampleManager.cs

```diff
@@ -98,19 +103,10 @@ private void RefreshSolverHandler()
             }
         }
 
-        private void DestroySolver()
+        private void DisableSolver()
         {
-            if (currentSolver != null)
-            {
-                DestroyImmediate(currentSolver);
-                currentSolver = null;
-            }
-
-            if (handler != null)
-            {
-                DestroyImmediate(handler);
-                handler = null;
-            }
+            foreach (Solver solver in GetComponents<Solver>())
```

Comment:
Consider using a for loop instead of foreach, for performance.

Reply:
This is something that we talked a bunch as a team about before, and in later version of unity/c# this isn't the same issue that it's been.

I'm happy to have this conversation separate from this PR to get numbers here, but I'm also not concerned because this isn't not a critical path (not update loop).